httomo/runner/loader.py

```diff
@@ -133,9 +134,12 @@ def __init__(
             angles=angles_arr,
             flats=flats_arr,
             darks=darks_arr,
-            global_index=self._chunk_index,
+            # Hardcoded for every MPI process because it's not being used in `FullFileDataSet`
+            # to offset block reading, that's what `data_offset` is being used for
+            global_index=(0, 0, 0),
```

Comment:
I'm not sure about that actually - shouldn't the `global_index` be the index of the chunk within the global data, i.e. non-zero for processes of rank 1 and above, and the `data_offset` be `(self._data_indices[0], 0, 0)` ?

How would `block.global_index` look like for process 1, after it calls `make_block(0, 0, 10)`? That block should have a non-zero `global_index`.